controllers/batch-deletion.js

```diff
@@ -0,0 +1,42 @@
+/*
+ * One Controller per layout view
+ */
+
+// const _ = require('lodash');
+const express = require("express");
+const multer = require('multer');
+const router = express.Router();
+const authHelper = require("../helpers/authentication");
+const { api } = require("../api");
+
+const storage = multer.memoryStorage();
+const upload = multer({ storage: storage });
+
+router.use(authHelper.authChecker);
+
+router.get("/", function (req, res, next) {
+  const deletionContainerHead = ["Schüler-IDs", "Lehrer-IDs"];
+
+  res.render("batch-deletion/batch-deletion", {
+    title: "Sammellöschung von Schülern",
+    user: res.locals.currentUser,
+    themeTitle: process.env.SC_NAV_TITLE || "Schul-Cloud",
+    deletionContainerHead,
+  });
+});
+
+router.post('/create-batch-deletion-file', upload.single('file'), (req, res, next) => {
+  if (req.file) {
+    console.log('Datei empfangen:', req.file.originalname);
+    const fileBuffer = req.file.buffer;
+    // TODO: Send the file to the API "schulcloud-server"
+    res.redirect(303, '/batch-deletion/');
+  } else {
+    res.status(400).send('Keine Datei hochgeladen');
+  }
+});
```

Comment:
When looking at the (already existing) upload code code for terms of use there might be a simpler solution that doesn't require to install a library.
https://github.com/hpi-schul-cloud/superhero-dashboard/blob/33812e826e3e9f7693a6b331d0508f5fafba7815/controllers/management/managementLogic.js#L24-L37
We could even transmit the CSV file "as is" because it is just text, hand it over to the server and parse the data in the server API.